packages/components/src/vue/components/morph-parts/inflections-list.vue

```diff
@@ -138,15 +170,29 @@
       }
     },
     computed: {
+      listClasses () {
+        const klasses = ["alpheios-inflections-list__inflections"]
+        klasses.push(`alpheios-inflections-list__${this.listclass}`)
+        return klasses.join(" ")
+      },
+
       hasInflections () {
         return this.inflections.length > 0
       },
 
       inflections () {
-        return (
-          this.$store.state.app.morphDataReady && this.app.hasMorphData() && this.lexeme.getGroupedInflections)
-          ? this.lexeme.getGroupedInflections()
-          : []
+        if (this.disambiguated) {
+          return (
+            this.$store.state.app.morphDataReady && this.app.hasMorphData() && this.lexeme.getSelectedInflectionForDisplay)
+            ? this.lexeme.getSelectedInflectionForDisplay()
+            : []
+
+        } else {
+          return (
+            this.$store.state.app.morphDataReady && this.app.hasMorphData() && this.lexeme.getGroupedInflections)
+            ? this.lexeme.getGroupedInflections()
+            : []
+        }
```

Comment:
eventually I think we should dedupe the selected inflection out of the grouped inflections, but this is too big a change for an incremental release.

For annotations, we're going to need to clean this up further, so that inflections can be annotated on distinct from the grouped view of them.